corehq/apps/change_feed/consumer/pillow.py

```diff
@@ -2,18 +2,21 @@
 from corehq.apps.change_feed import topics
 from corehq.apps.change_feed.consumer.feed import KafkaChangeFeed
 from pillowtop.checkpoints.manager import PillowCheckpoint, PillowCheckpointEventHandler
-from pillowtop.dao.couch import CouchDocumentStore
+from pillowtop.dao.django import DjangoDocumentStore
+from pillowtop.models import DjangoPillowCheckpoint
 from pillowtop.pillow.interface import ConstructedPillow
 from pillowtop.processor import NoopProcessor
 
 
 # this number intentionally left high to avoid many redundant saves while this pillow is still
 # in experimental stage
-KAFKA_CHECKPOINT_FREQUENCY = 10000
+KAFKA_CHECKPOINT_FREQUENCY = 1000
 
 
 def get_demo_case_consumer_pillow():
-    document_store = CouchDocumentStore(CommCareCase.get_db())
+    document_store = DjangoDocumentStore(
+        DjangoPillowCheckpoint, DjangoPillowCheckpoint.to_dict, DjangoPillowCheckpoint.from_dict,
+    )
```

Comment:
Instantiating the doc store this way means I can only be used for checkpoints right? Maybe call it `checkpoint_doc_store` or some such things to make it more clear.